.bazelci/presubmit.yml

```diff
@@ -21,6 +21,11 @@ tasks:
     - "--test_tag_filters=-fix-bazelci-ubuntu"
     test_targets:
     - "//..."
+  ubuntu1604_angular_bazel_example:
+    name: ubuntu1604_angular_bazel_example
+    platform: ubuntu1604
+    test_targets:
+    - "//:e2e_angular_bazel_example"
```

Comment:
why is it separate? what happens when we move angular_bazel_example to the examples/ directory - then will it be part of the normal test targets?

Reply:
running into memory issues running such a large test target with bazel in bazel so had to separate it out for now. may need to limit the memory & number of parallel actions for the nested bazel. still investigating.